.github/workflows/ci.yml

```diff
@@ -12,12 +19,19 @@ jobs:
         with:
           java-version: 8
 
-      - name: Cache Maven packages
+      - name: Cache local Maven repository
         uses: actions/cache@v2
         with:
-          path: ~/.m2
-          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
-          restore-keys: ${{ runner.os }}-m2
+          path: ~/.m2/repository
```

Comment:
Little functional change here, just matching the example in the [docs](https://github.com/actions/cache/blob/main/examples.md#java---maven)